notebook/static/base/js/namespace.js

```diff
@@ -73,7 +73,7 @@ define(function(){
     // tree
     jglobal('SessionList','tree/js/sessionlist');
 
-    Jupyter.version = "5.2.1";
+    Jupyter.version = "5.3.0.dev0";
```

Comment:
@agrippa I think version changes automatically when start developing the code base. And should be changed when we are making a PR, if I'm not mistaken.
I'm currently new to the codebase so I might be wrong in this.  😄